### ISSUE TITLE
show always

### ISSUE THREAD

**Issue by mdre** [pzuraq/vscode-vertical-tab-bar#4]
I could not get it to work on VSCode 1.39.2
But is there any way to show the tabs always but on the right? I have a wide screen with enough space to show the sidebar and put the tabs on the right.
Thanks.

**Comment by martin-braun** [comment]
I agree, this extension needs some options like "display on right" and "show always". All this extension does now is supplying an CSS and we use a different extension to hook it in. To have such options, the extension would require to inject the CSS by itself, or alternatively, modify the DOM by adding some class names for specific options, to get the desired result out of it.

In either case, the extension will need an overhaul.
